website_sale_hide_discount/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==========================
+Website Sale Hide Discount
+==========================
+
+Only show the pricelist price in website sale, instead of showing the original
+price crossed out.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/50
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/e-commerce/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```

Comment:
Last time I talked about this, we came to the conclusion that self-crediting was not required, since OCA is already an author of all addons, so it is expected that all code & assets are automatically credited to them if not stated otherwise. This was in the template as a placeholder, just in case you use external sources for images. Some of them have CC licenses that require crediting for instance.

In any case, it does not hurt if you prefer to keep it 😉